R/gauge.R

```diff
@@ -61,7 +61,27 @@ gauge <- function(value, min, max, sectors = gaugeSectors(),
     name = 'gauge',
     x,
     package = 'flexdashboard',
-    dependencies = rmarkdown::html_dependency_jquery()
+    dependencies = rmarkdown::html_dependency_jquery(),
+    preRenderHook = function(widget) {
+      theme <- bslib::bs_current_theme()
```

Comment:
This will cause Shiny to be loaded, but I think that's a bug in bslib. See https://github.com/rstudio/bslib/issues/261